### PR TITLE
USWDS-3475 Combo Box Dispatch Change Event

### LINKED ISSUE #3475: Combo Box component does not fire a change event on the select when the value is changed
## Description

When changing the selection for the new Combo Box component, I expect a proper `change` event to be fired on the `<select>`. Currently no event is fired when USWDS programmatically sets the `value` property for the `<select>`.

## Steps to reproduce the issue

1. Copy the example code for a combobox from [here](https://designsystem.digital.gov/components/form-controls/#combo-box)
2. Add an event listener to the select:
```javascript
document.getElementById('fruit').addEventListener('change', (event) => {
  console.log(event);
});
```
3. Run the code and select an option in the combo box
4. No `change` event is fired so no event is logged to the console

## Additional information [optional]

~Unrelated, but the combo box also does not respect `disabled` or `readonly` attributes on the `<select>`.~ I see the following issue was created for this: #3474.

### PR DESCRIPTION
**Pull request by wbyoko**
* trigger a change event on input and select change

closes #3475

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Comment by wbyoko** [comment]
@ConnorDY Does this seem sufficient? This will emit change events for both the input and the select.

**Comment by ConnorDY** [comment]
> @ConnorDY Does this seem sufficient? This will emit change events for both the input and the select. 

Looks great besides the one comment I left.

**Comment by wbyoko** [comment]
@ConnorDY I don't see a comment anywhere. What comment are you referencing?

**Comment by ConnorDY** [comment]
> @ConnorDY I don't see a comment anywhere. What comment are you referencing? 

Sorry. The new GitHub mobile app is a bit confusing. The comment was left pending but now it has been submitted.

File: src/js/components/combo-box.js [hunk 1]
```diff
@@ -19,6 +19,21 @@ const LIST_OPTION = `.${LIST_OPTION_CLASS}`;
...
+ * set the value of the element and dispatch a change event
+ *
+ * @param {HTMLInputElement|HTMLSelectElement} el The element to update
+ * @param {string} value The new value of the element
+ */
+const changeElementValue = (el, value = "") => {
+  const elementToChange = el;
+  elementToChange.value = value;
+
+  const event = document.createEvent("Event");
```
[the remark this answers is not in the document]
Comment (wbyoko) [reply src/js/components/combo-box.js:32]: We are using `document.createEvent` at this point for ie11 compatibility, which does not support the `Event` constructor.

[the remark this answers is not in the document]
Comment (wbyoko) [reply src/js/components/combo-box.js:32]: I'll add a polyfill for ie11 and update this. Thanks!

[the remark this answers is not in the document]
Comment (wbyoko) [reply src/js/components/combo-box.js]: @ConnerDY I've added the polyfill. Does this work?

[the remark this answers is not in the document]
Comment (ConnorDY) [reply src/js/components/combo-box.js]: @wbyoko Looks great based on the tests. Unable to test this out myself as I am out of town currently.

**Pull request merged**